pkg/services/reputation/common/router/calls.go

```diff
@@ -94,7 +94,7 @@ func (w *trustWriter) Write(t reputation.Trust) error {
 				continue
 			}
 
-			remoteWriter, err = provider.InitWriter(w.ctx)
+			remoteWriter, err = provider.InitWriter(w.ctx.Context)
```

Comment:
Just `w.ctx`

Reply:
added commentary why this is necessary and `TODO` to think if the router can be done general without context cast